jobs/kafka-exporter/templates/ctl.erb

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+RUN_DIR=/var/vcap/sys/run/kafka-exporter
+LOG_DIR=/var/vcap/sys/log/kafka-exporter
+LOG_STDOUT=$LOG_DIR/kafka-exporter.stdout.log
+CONF_DIR=/var/vcap/jobs/kafka-exporter/config
+CERTS_DIR=/var/vcap/jobs/kafka-exporter/certs
+PKG_DIR=/var/vcap/packages/kafka-exporter
+JOBS_DIR=/var/vcap/jobs/kafka-exporter
+PIDFILE=$RUN_DIR/kafka-exporter.pid
+
+case $1 in
+
+  start) 
```

Comment:
can we maybe adjust this section to follow the approach we have here? 

https://github.com/cloudfoundry/prometheus-boshrelease/blob/0273270982fb47600a4ed8cb87b794cb77888ded/jobs/prometheus/templates/bin/prometheus_ctl#L39